MAVProxy/tools/MAVExplorer.py

```diff
@@ -428,9 +429,13 @@ def cmd_param(args):
 def cmd_loadfile(args):
     '''callback from menu to load a log file'''
     if len(args) != 1:
-        print "Error loading file"
+        fileargs = " ".join(args)
```

Comment:
Should we allow quoting of the filename here instead?  Use shlex to break the line up for us?  Taking logically separate arguments and squishing them together is a little awkward.